### PR TITLE
Package request python-fenics-dolfinx

### PR DESCRIPTION
**Pull request by carlosal1015**
I think that the package [python-fenics-dolfinx](https://aur.archlinux.org/packages/python-fenics-dolfinx) will adopt more users [1-2] and fits in the context of mathematics (partial differential equations). It using as dependency `petsc-complex`, `parmetis` that are already in `arch4edu`.

Running the following commands

```console
$ python smart-add-package.py python-fenics-dolfinx x86_64
$ rm -r x86_64/petsc
```
Appending these lines to `x86_64/dolfinx/cactus.yaml`
```yml
- source: archpkg
    archpkg: boost
```

- [1] https://github.com/FEniCS/dolfinx/issues?q=archlinux+
- [2] https://github.com/FEniCS/dolfinx/issues?q=arch+linux+

Thanks.

### CODE REVIEW

File: x86_64/python-fenics-ufl/cactus.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+../../template/x86_64-simple.yaml
```
Comment (petronny) [inline x86_64/python-fenics-ufl/cactus.yaml:1]: This one should use the x86_64-python template.

File: x86_64/python-fenics-ffcx/cactus.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+nvchecker:
+  - source: aur
+    aur:
```
Comment (petronny) [inline x86_64/python-fenics-ffcx/cactus.yaml:3]: Should also have the python alias upstream. And so do the other python packages.

**Review by petronny — CHANGES_REQUESTED** [review]

File: x86_64/python-fenics-ufl/cactus.yaml (hunk 1 above)
Comment (carlosal1015) [reply x86_64/python-fenics-ufl/cactus.yaml]: Thanks, done with `ln -s ../../template/x86_64-python cactus.yaml`.

File: x86_64/python-fenics-ffcx/cactus.yaml (hunk 2 above)
Comment (carlosal1015) [reply x86_64/python-fenics-ffcx/cactus.yaml:3]: Thanks, now include python alias.

**Review by petronny — APPROVED** [review]

**Pull request merged**